scipy/optimize/_highs/meson.build

```diff
@@ -51,171 +56,196 @@ basiclu_lib = static_library('basiclu',
-  '../../_lib/highs/src/parallel/HighsTaskExecutor.cpp',
-  '../../_lib/highs/src/presolve/ICrashX.cpp',
-  '../../_lib/highs/src/presolve/HAggregator.cpp',
-  '../../_lib/highs/src/presolve/HighsLpPropagator.cpp',
-  '../../_lib/highs/src/presolve/HighsPostsolveStack.cpp',
-  '../../_lib/highs/src/presolve/HighsSymmetry.cpp',
-  '../../_lib/highs/src/presolve/HPreData.cpp',
-  '../../_lib/highs/src/presolve/HPresolve.cpp',
-  '../../_lib/highs/src/presolve/PresolveAnalysis.cpp',
-  '../../_lib/highs/src/presolve/PresolveComponent.cpp',
-  '../../_lib/highs/src/presolve/Presolve.cpp',
-  '../../_lib/highs/src/presolve/PresolveUtils.cpp',
-  '../../_lib/highs/src/qpsolver/basis.cpp',
-  '../../_lib/highs/src/qpsolver/solver.cpp',
-  '../../_lib/highs/src/qpsolver/ratiotest.cpp',
-  '../../_lib/highs/src/simplex/HEkk.cpp',
-  '../../_lib/highs/src/simplex/HEkkControl.cpp',
-  '../../_lib/highs/src/simplex/HEkkDebug.cpp',
-  '../../_lib/highs/src/simplex/HEkkPrimal.cpp',
-  '../../_lib/highs/src/simplex/HEkkDual.cpp',
-  '../../_lib/highs/src/simplex/HEkkDualRHS.cpp',
-  '../../_lib/highs/src/simplex/HEkkDualRow.cpp',
-  '../../_lib/highs/src/simplex/HEkkDualMulti.cpp',
-  '../../_lib/highs/src/simplex/HEkkInterface.cpp',
-  '../../_lib/highs/src/simplex/HighsSimplexAnalysis.cpp',
-  '../../_lib/highs/src/simplex/HSimplex.cpp',
-  '../../_lib/highs/src/simplex/HSimplexDebug.cpp',
-  '../../_lib/highs/src/simplex/HSimplexNla.cpp',
-  '../../_lib/highs/src/simplex/HSimplexNlaDebug.cpp',
-  '../../_lib/highs/src/simplex/HSimplexNlaFreeze.cpp',
-  '../../_lib/highs/src/simplex/HSimplexNlaProductForm.cpp',
-  '../../_lib/highs/src/simplex/HSimplexReport.cpp',
-  '../../_lib/highs/src/test/DevKkt.cpp',
-  '../../_lib/highs/src/test/KktCh2.cpp',
-  '../../_lib/highs/src/util/HFactor.cpp',
-  '../../_lib/highs/src/util/HFactorDebug.cpp',
-  '../../_lib/highs/src/util/HFactorExtend.cpp',
-  '../../_lib/highs/src/util/HFactorRefactor.cpp',
-  '../../_lib/highs/src/util/HFactorUtils.cpp',
-  '../../_lib/highs/src/util/HighsHash.cpp',
-  '../../_lib/highs/src/util/HighsLinearSumBounds.cpp',
-  '../../_lib/highs/src/util/HighsMatrixPic.cpp',
-  '../../_lib/highs/src/util/HighsMatrixUtils.cpp',
-  '../../_lib/highs/src/util/HighsSort.cpp',
-  '../../_lib/highs/src/util/HighsSparseMatrix.cpp',
-  '../../_lib/highs/src/util/HighsUtils.cpp',
-  '../../_lib/highs/src/util/HSet.cpp',
-  '../../_lib/highs/src/util/HVectorBase.cpp',
-  '../../_lib/highs/src/util/stringutil.cpp',
-  '../../_lib/highs/src/interfaces/highs_c_api.cpp'
-]
 
 if cc.has_argument('-Wno-unused-but-set-variable')
   Wno_unused_but_set = ['-Wno-unused-but-set-variable']
 else
   Wno_unused_but_set = []
 endif
 
-_highs_wrapper = py3.extension_module('_highs_wrapper',
-  ['cython/src/_highs_wrapper.pyx', highs_sources, ipx_sources],
+highs_flags = ['-Wno-sign-compare', '-Wno-switch', '-Wno-format-truncation', '-Wno-non-virtual-dtor', '-Wno-class-memaccess']
```

Comment:
Another case of putting in a list to apply to multiple libraries -- please refactor if desired